content/en/docs/setup/release/notes.md

```diff
@@ -190,7 +190,7 @@ Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provi
   PodSubnet validates against the corresponding cluster "--node-cidr-mask-size" of the kube-controller-manager, it fail if the values are not compatible.
   kubeadm no longer sets the node-mask automatically on IPv6 deployments, you must check that your IPv6 service subnet mask is compatible with the default node mask /64 or set it accordenly. 
   Previously, for IPv6, if the podSubnet had a mask lower than /112, kubeadm calculated a node-mask to be multiple of eight and splitting the available bits to maximise the number used for nodes. ([#95723](https://github.com/kubernetes/kubernetes/pull/95723), [@aojea](https://github.com/aojea)) [SIG Cluster Lifecycle]
-- The deprecated flag --experimental-kustomize is now removed from kubeadm commands. Use --experimental-patches instead, which was introduced in 1.19. Migration infromation available in --help description for --exprimental-patches. ([#94871](https://github.com/kubernetes/kubernetes/pull/94871), [@neolit123](https://github.com/neolit123))
+- The deprecated flag --experimental-kustomize is now removed from kubeadm commands. Use --experimental-patches instead, which was introduced in 1.19. Migration infromation available in --help description for --experimental-patches. ([#94871](https://github.com/kubernetes/kubernetes/pull/94871), [@neolit123](https://github.com/neolit123))
```

Comment:
```suggestion
- The deprecated flag --experimental-kustomize is now removed from kubeadm commands. Use --experimental-patches instead, which was introduced in 1.19. Migration information available in --help description for --experimental-patches. ([#94871](https://github.com/kubernetes/kubernetes/pull/94871), [@neolit123](https://github.com/neolit123))
```

Reply:
Okay

Reply:
Hi @Tej-Singh-Rana . Thanks for opening this pull request.
The changes look good to me.
What do you think about accepting the suggestion from @sftim?

Reply:
I already accepted his suggestion. @kbhawkey I have to do something? Please assist.

Reply:
Hi @Tej-Singh-Rana . I am looking at the changes and suggestions.
There is another spelling error ("Migration information ... ") that needs fixing.
Do you have time to fix, otherwise, this PR can move forward without incorporating that fix.

Reply:
Note: This file is auto-generated at release time.

Reply:
Hello, @kbhawkey done.

Reply:
Thanks @Tej-Singh-Rana !
/lgtm